graphcanvas/graph_view.py

```diff
@@ -115,9 +119,9 @@ def _layout_changed(self, new):
         self._canvas.style = new
 
     def _on_hover(self, label):
-        print "hovering over:", label
+        print(u"hovering over: {}".format(text(label)))
```

Comment:
It should be `unicode`, as it is a `Unicode` trait here: https://github.com/enthought/graphcanvas/pull/11/files#diff-66da25d11c6dbb1b56202ecf13a9c6b1R25